### DISCUSSION TITLE
Separate cabin lights from dome lights (and maybe improve them)

### DISCUSSION THREAD

**Discussion by ZaccJacc** [magknight/787aviatorsTracker#843] [Ideas]
Could the cabin lights be separated from the dome lights? Bit annoying when the cabin lights should be on mid-cruise on a night flight but dome shouldn't be on full, it breaks the immersion somewhat.
The cabin lighting is also a little iffy in the sense that it doesn't actually light the cabin up when its dark that well. The ceiling will have a colour on it but the actual cabin remains in darkness despite the little lights under the overhead bins. Could this be improved so the ambient light is greater?
Maybe the lights could have 2 phases (just ceiling, and then ceiling + ambient) and be controlled in a tab in the efb (assuming there isn't enough panel space/time and willpower to add a custom page onto the ipad to control it)
Thanks :)

**Comment by aeroniemi** [comment]
lights aren't a particularly easy thing to change because they're a pain to get working right -we might change it if we ever do cabin upgrades, but that isn't likely
